backend/logic/deploy/bucket/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Bucket deployment module
+ *
+ * Sub-Modules
+ * ===========
+ * Sub-modules implement different providers (e.g. GCP, AWS). Every bucket
+ * deployment module in this directory should conform to this minimum API:
+ *
+ * - isAvailable({ networkConfig }) -> bool - If the module can be used
+ * - configure({ networkConfig }) - Called to allow the module to configure its singleton
+ * - deploy({ networkConfig, OutputDir }) -> BucketDeployObject - Deploy OutputDir to a bucket
+ *
+ * BucketDeployObject
+ * ==================
+ * {
+ *    bucketName: "mybucket", // the name of the bucket deployed to
+ *    url: "gs://mybucket/",
+ *    httpUrl: "https://storage.googleapis.com/mybucket/"
+ * }
+ */
+const { getModules, loadModule } = require('../../../utils/module')
+const { getLogger } = require('../../../utils/logger')
+
+const log = getLogger('logic.deploy.bucket')
+
+/**
+ * Deploy shop to a bucket
```

Comment:
nit: it seems it could upload to multiple buckets if more than 1 module is configured?